upload.py

```diff
@@ -126,6 +134,14 @@ def main():
             upload_emoji(session, emoji_name, filename)
             print("{} upload complete.".format(filename))
             uploaded += 1
+
+    for slackmoji_file in args.slackmoji_files:
+        if os.path.isdir(slackmoji_file):
```

Comment:
If I'm reading this right, this stops traversal at a single directory deep. Any reason not to walk the filesystem recursively - maybe guarded by a max-depth flag or something?

Reply:
Sure, mebbe could change this to an os.walk instead - this worked for my use case where all the emojis were in one directory (and I suspect is how almost everybody has their emoji exported, because everyone's using [that one emoji downloader](https://gist.github.com/lmarkus/8722f56baf8c47045621))